app/frontend/packs/degree-type-autocomplete.js

```diff
@@ -0,0 +1,48 @@
+import accessibleAutocomplete from "accessible-autocomplete";
+
+const initDegreeTypeAutocomplete = () => {
+  try {
+    const inputId = "degree-type";
+    const input = document.getElementById(inputId);
+    const containerId = "degree-type-autocomplete";
+    const container = document.getElementById(containerId);
+    const sourceData = JSON.parse(container.dataset.source);
+
+    function inputTemplate(result) {
+      if (result) {
+        const descriptor = result.split('|');
+
+        return descriptor[1]
+      }
+    }
+
+    function suggestionTemplate(result) {
+      const descriptor = result.split('|');
+
+      if (descriptor[0]) {
+        return `<strong>${descriptor[0]}</strong> <span class="autocomplete__option--hint">${descriptor[1]}</span>`
+      }
+      return `<strong>${descriptor[1]}</strong>`
+    }
+
+    if (!container) return;
+
+    input.remove();
```

Comment:
You should do this after the `accessibleAutocomplete` call; if for whatever reason the JS fails below, the user will be left without an input!